content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Configuring Tracing Libraries Using OpenTelemetry Environment Variables
+further_reading:
+    - link: '/tracing/trace_collection/library_config/dotnet-core'
+      tag: 'Documentation'
+      text: '.NET Core Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/dotnet-framework'
+      tag: 'Documentation'
+      text: '.NET Framework Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/go'
+      tag: 'Documentation'
+      text: 'Go Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby Tracing Configuration'
+---
+
+After you set up the tracing library with your code and configure the Agent to collect APM data, optionally configure the tracing library as desired using OpenTelemetry environment variables
+
+Tracer settings can be configured with the following environment variables to their equivalent Datadog environment variable mapping:
+
+`OTEL_SERVICE_NAME`
+: **Mapping**: `DD_SERVICE`<br>
```

Comment:
I think we may want to add a text-based description in case the user isn't familiar with any of our configuration knobs. Maybe something like this could be reworked for all of the other configurations being added to this page? BTW I'm getting inspiration from the [OpenTelemetry docs](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration):

```suggestion
: **Corresponding Datadog Environment Variable**: `DD_SERVICE`<br>
Sets the service name.<br>
**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence.
```